docs/api.md

```diff
@@ -171,7 +171,34 @@ spec:
 
   # Optional: If true, the GitOps Service will allow Argo CD to connect to the specified cluster
   # even if it is using an invalid or self-signed TLS certificate.
+  # Defaults to false.
   allowInsecureSkipTLSVerify: false
+
+  # Optional: Controls whether Argo CD will use the ServiceAccount provided by the user in the Secret, or if a new ServiceAccount
+  # should be created.
+  # 
+  # - If true, the GitOps Service will automatically create a ServiceAccount/ClusterRole/ClusterRoleBinding on the target cluster,
+  #   using the credentials provided by the user in the secret. 
+  #   - Argo CD will then be configured to deploy with that new ServiceAccount.
+  #
+  # - Default: If false, it is assumed that the credentials provided by the user in the Secret are for a ServiceAccount on the cluster, and
+  #   Argo CD will be configred to use the ServiceAccount referenced by the Secret of the user. No new ServiceAccount will be created.
+  #   - This should be used, for example, when the ServiceAccount Argo CD is using will not full cluster access (*/*/* at cluster scope)
```

Comment:
`...will not full cluster access...` should be `...will not have full cluster access...`. It might read better to say `...does not have full cluster access...`